docs/pages/guides/prebuild.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Prebuild
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+Before a native app can be compiled, the native source code must be generated. Expo CLI provides a unique and powerful system called _prebuilding_, which generates the native code for your project based on four factors:
+
+1. The [Expo Config][expo-config] file (`app.json`, `app.config.js`).
+2. Arguments passed to the `npx expo prebuild` command.
+3. Version of `expo` that's installed in the project.
+4. Expo [Autolinking](/workflow/glossary-of-terms#autolinking), which finds native modules in your `package.json` and automatically links them as CocoaPods on iOS and Gradle packages on Android.
+
```

Comment:
```suggestion

Prebuild is run automatically when you build a managed Expo project on EAS Build ([learn more](https://docs.expo.dev/build-reference/ios-builds/)), and it's not necessary to ever run it on your local machine, but there may be cases where you want to. For example, if you run `npx expo run:[ios|android]`, Expo CLI will run prebuild for you. This is helpful to debug your project locally in the respective native IDE when you're unable to diagnose the issue other ways. You may also invoke prebuild directly in order to test your config plugins or work on native modules. This guide explains how to use prebuild, and why you might or might not want to use it.

```

Reply:
The purpose of this doc is to eliminate confusing phrases like "managed Expo project".

> and it's not necessary to ever run it on your local machine, but there may be cases where you want to.

We probably want to avoid these dilemma like phrases, it should be no big deal to run prebuild.

Reply:
I split this across the usage secion